.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/local/bin/python3"
```

Comment:
remove this file - it's likely that other people will have their own `.vscode` settings, and/or Python in a different location.